templates/commands/render/action.go

```diff
@@ -151,6 +181,15 @@ func processPaths(paths []model.String, scope *common.Scope) ([]model.String, er
 		if err != nil {
 			return nil, err
 		}
+
+		// on Windows, escaping is disabled in glob parsing. Therefore, to keep
+		// glob parsing behavior consistent across different OSes, escaping is
+		// not permitted. This is checked in processPaths because FromSlash converts
+		// separators to backslashes for Windows.
+		if strings.Contains(p.Val, `\`) {
+			return nil, p.Pos.Errorf(`escaping glob paths is not permitted: %q`, p.Val)
```

Comment:
Maybe we can make this error message a little more helpful by adding the word "backslashes" somewhere, like "backslashes aren't allowed in path globs"

Reply:
Added backslashes to error message.